tests/utils.py

```diff
@@ -6,7 +6,7 @@
 from datahub.ingestion.api.common import EndOfStream, PipelineContext, RecordEnvelope
 from datahub.ingestion.transformer.dataset_transformer import DatasetTransformer
 from datahub.metadata.schema_classes import MetadataChangeEventClass
-from datahub.utilities.urns.urn import Urn
+from datahub.utilities.urns._urn_base import Urn
```

Comment:
why are you importing from this private module? What's the difference between this `Urn` and `urns.urn.Urn`?

Reply:
This fixes a type error for me (via `"python.analysis.typeCheckingMode": "basic"` in vscode)
> "Urn" is not exported from module "datahub.utilities.urns.urn"
  Import from "datahub.utilities.urns._urn_base" insteadPylance[[reportPrivateImportUsage]](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportPrivateImportUsage)